html/changelog.html

```diff
@@ -684,36 +684,6 @@ <h3 class="author">maikilangiolo updated:</h3>
 				<li class="rscadd">Added lights to theater storage</li>
 				<li class="tweak">reworked entirety of club layout: more open, better looking.</li>
 			</ul>
```

Comment:
```suggestion
			</ul>
			
			<h2 class="date">23 July 2022</h2>
			<h3 class="author">MemedHams updated:</h3>
			<ul class="changes bgimages16">
				<li class="rscadd">nobles who spawn as part of Neotheology now get a 50% chance of aquiring a Neotheo heirloom</li>
				<li class="tweak">more items for the basic heirloom pool</li>
				<li class="tweak">noble heirloom names are now visibly structured "(item) of (family)"</li>
				<li class="tweak">noble heirlooms are now considerably more expensive. Watch your pockets!</li>
			</ul>
			<h3 class="author">assortedbeads updated:</h3>
			<ul class="changes bgimages16">
				<li class="tweak">Recipe for crossbow bolts moved to guns section, right under powered crossbow.</li>
				<li class="tweak">Crossbow recipe edited slightly to include glass, representing a poor scope.</li>
				<li class="tweak">Crossbow has a minor zoom factor now, 3 spaces when used.</li>
				<li class="tweak">Bolts now embed on hit and can be extracted from kills in bolt form to be used again. Prior to this they turned into metal rods on hits.</li>
				<li class="tweak">Embedded bolts are sharp, dealing pain, brute and toxin like normal shrapnel if you run without removing them. They can be removed via surgery or right clicking to yank.</li>
				<li class="tweak">Recipe for bolts costs the same, but you now spend 5 metal rods to make a handful of 5 bolts rather than 5 individual bolts, less clicks on players side.</li>
				<li class="tweak">You now get chat messages telling you when you load or unload a bolt to/from the crossbow.</li>
				<li class="tweak">Remaining bolt stack sprite properly updates when loading the crossbow with a single one.</li>
				<li class="tweak">You now only unload a bolt from the crossbow if you interact with an empty hand on a loaded crossbow you are currently holding.</li>
				<li class="balance">Energy use of powered crossbow was made more efficient, 50 instead of 100.</li>
				<li class="balance">Fire Delay of powered crossbow was lowered, 6 instead of 12.</li>
				<li class="balance">Projectile damage of powered crossbow was buffed, 40 -> 55.</li>
				<li class="balance">Projectile penetration of powered crossbow was buffed 20 -> 30.</li>
				<li class="imageadd">Bolt projectile sprite was changed to something more fitting</li>
			</ul>
			<h3 class="author">komunre updated:</h3>
			<ul class="changes bgimages16">
				<li class="rscadd">Rubber pig that oinks</li>
			</ul>
```
No reason to modify changelog manually